local-cluster/src/local_cluster_snapshot_utils.rs

```diff
@@ -0,0 +1,115 @@
+use crate::{cluster::Cluster, local_cluster::LocalCluster};
+use log::*;
+use solana_runtime::{
+    snapshot_archive_info::{
+        FullSnapshotArchiveInfo, IncrementalSnapshotArchiveInfo, SnapshotArchiveInfoGetter,
+    },
+    snapshot_utils,
+};
+use solana_sdk::{client::SyncClient, commitment_config::CommitmentConfig};
+use std::{path::Path, thread::sleep, time::Duration};
+
+impl LocalCluster {
+    /// Return the next full snapshot archive info after the cluster's last processed slot
+    pub fn wait_for_next_full_snapshot(
+        cluster: &LocalCluster,
```

Comment:
I think these can be replaced with `&self`

Reply:
Done!